README.md

```diff
@@ -1,3 +1,9 @@
+```diff
+- This is a DevOps ephemeral branch for configuring the CI/CD in AWS. There are no new features or code. 
+- Nothing to see here. To use GodMode-Ganache-CLI, the Develop branch is where you want to be
```

Comment:
If needed let's just make this more clear like, `Use this library from `develop` branch`